MS DNS Winlogbeat Mapping (#28975)
demisto/content: Packs/MicrosoftDNS/README.md

You will need to configure the vendor and product for this specific collector.

### XDRC (XDR Collector)
Use the information described [here](https://docs.paloaltonetworks.com/cortex/cortex-xdr/cortex-xdr-pro-admin/cortex-xdr-collectors/xdr-collector-datasets#id7f0fcd4d-b019-4959-a43a-40b03db8a8b2).

You can configure the vendor and product by replacing [vendor]\_[product]\_raw with *microsoft_dns_raw*.

When configuring the instance, you should use a yml file that configures the vendor and product, as shown in the below configuration for the Microsoft DNS product.

**Pay Attention**:

> When using this pack their are two integrations available for it.

Reviewer: ```suggestion
When using this pack there are two integrations available for it.
```

* ***As enrichment, forwarding DNS Audit logs is supported via Winlogbeat***

The first, is via Filebeat for DNS Debug logs.
The second, is via Winlogbeat for DNS Audit logs.

Follow these steps in order to configure the XDR Collector:
1. The implementation of the Collector for both of the methods requires to create a [Profile](https://docs-cortex.paloaltonetworks.com/r/Cortex-XDR/Cortex-XDR-Pro-Administrator-Guide/Add-an-XDR-Collector-Profile-for-Windows) for each integration.

>    * For **XSIAM version 1.2** only, copy and paste the below in the *Filebeat Configuration File* section (inside the relevant profile under the *XDR Collectors Profiles*).

Reviewer: ```suggestion
   * For **XSIAM version 1.2** only, in the relevant profile under the *XDR Collectors Profiles*, copy and paste the information from the [Filebeat Configuration File](#filebeat-configuration-file) section.
```

```
**Note**: The above configuration uses the default location of the logs. 

### Winlogbeat Configuration File

```
winlogbeat.event_logs: 
  - name: Microsoft-Windows-DNSServer/Audit
    processors: 
      - add_fields: 
          fields: 
            vendor: microsoft
            product: dns
    id: dns-audit-logs
```

> For **XSIAM version 1.3** and above, please use the built-in YAML template provided within the XDR collector configuration.

Reviewer: ```suggestion
For **XSIAM version 1.3** and above, use the built-in YAML template provided within the XDR collector configuration.
```
